var/spack/repos/builtin/packages/arbor/package.py

```diff
@@ -6,68 +6,68 @@
 from spack import *
 
 
-class Arbor(CMakePackage):
+class Arbor(CMakePackage, CudaPackage):
     """Arbor is a high-performance library for computational neuroscience
     simulations."""
 
     homepage = "https://github.com/arbor-sim/arbor/"
-    url      = "https://github.com/arbor-sim/arbor/archive/v0.2.tar.gz"
+    git      = "https://github.com/arbor-sim/arbor.git"
+    maintainers = ['bcumming', 'halfflat']
 
-    version('0.4', sha256='7d9fc6b3262954cc5dc1751215fbb9f2cdb7010e829a2be43022f90da2d8d2e3')
-    version('0.2', sha256='43c9181c03be5f3c9820b2b50592d7b41344f37e1200980119ad347eb7bcf4eb')
+    version('master', branch='master')
+    version('0.5', 'd0c8a4c7f97565d7c30493c66249be794d1dc424de266fc79cecbbf0e313df59',
+            url='https://github.com/arbor-sim/arbor/releases/download/v0.5/arbor-v0.5-full.tar.gz')
```

Comment:
Can you move `url=...` to the level of the class itself? It's OK to keep both `git` and `url`, the former is useful for building a dev versions and the latter will help to autodetect the url for later versions :)